modin/core/dataframe/pandas/dataframe/dataframe.py

```diff
@@ -1686,45 +1632,16 @@ def rename(
         -----
         If level is not specified, the default behavior is to replace row labels in all levels.
         """
```

Comment:
probably should remove this note as we removed the `level` parameter

Reply:
Removed

Reply:
@dchigarev ready for review